api/src/main/java/io/strimzi/api/kafka/model/KafkaMirrorMakerConsumer.java

```diff
@@ -0,0 +1,125 @@
+/*
```

Comment:
We don't need this class. The `KafkaMirrorMakerConsumer` is not a custom resource itself. We should just have the corresponding spec in order to use it as a field in the `KafkaMirrorMaker`